intelligence-services/latest/using/index.md

```diff
@@ -45,10 +45,10 @@ Follow these steps to use the default (i.e. out-of-the-box) configuration of Int
         For example, you can add any of the default renditions:
 
         ```bash
-        aiFeatures, aiLabels, aiTextract, aiSpeechToText, webvtt, aiPiiEntities 
+        aiFeatures, aiLabels, aiTextract
```

Comment:
We should list all default renditions, so remove this diff.

Should be:

aiFeatures, aiLabels, aiTextract, aiSpeechToText, webvtt, aiPiiEntities